### ISSUE TITLE
about the first batch norm

### ISSUE THREAD

**Issue by twmht** [miaow1988/ShuffleNet_V2_pytorch_caffe#28]
the first batch normalization layer is used to replace the mean subtract, but it should be used with `affine=False`.

this might be the issue that you can't reproduce the paper result.

**Comment by miaow1988** [comment]
I've tried that, but I still got similar results.
I've also tried use conventional normalization, mean = [0.485, 0.456, 0.406], std = [0.229, 0.224, 0.225]. The result is worse then using BN layer.

I think my problem is in data augmentation. But if you checked the PyTorch's official pretrianed modes, you'll find that the results of ShuffleNet V2 and MobileNet V2 is about 70% Acc. (https://pytorch.org/docs/stable/torchvision/models.html#classification)
This is similar to my experiments on ShuffleNet V2 and MobileNet V2. 

So I consider this is the problem of PyTorch itself. I think there are several differences between TensorFlow, PyTorch and Caffe:
1. the parameter of BN layers.
2. the application of weight decay.
3. etc...
